docs/build/guides/transactions/send-and-receive-payments.mdx

```diff
@@ -6,1405 +6,574 @@ sidebar_position: 7
+  - If you want to make a payment of a custom contract token which is not a Stellar asset but follows the [SEP-41 token standard](/docs/tokens/stellar-asset-contract#overview), you must use the token's contract. Stellar's payment operations can only be used to transfer Stellar assets.
 
 To learn more about the differences between Stellar assets and contract tokens, see the [Tokens](../../../../docs/tokens) overview.
 
-Additionally, there are two main options to interact with the Stellar network: the [Horizon API](../../../data/apis/horizon/README.mdx) and [Stellar RPC](../../../data/apis/rpc/README.mdx). Generally speaking, you should use RPC unless you need features that only Horizon provides. If your use case is limited to making payments, RPC should be sufficient. Regardless, each payment example will be broken down to show how to perform such payments with either option.
-
 <Alert>
 
-In the following code samples, proper error checking is omitted for brevity. However, you should _always_ validate your results, as there are many ways that requests can fail. You should refer to the guide on [Error Handling](../../../data/apis/horizon/api-reference/errors/error-handling.mdx) for tips on error management strategies.
+In the following code samples, proper error checking is omitted for brevity. However, you should _always_ validate your results, as there are many ways that requests can fail.
 
 </Alert>
 
-## Using the Payment Operation
+## Using Payments Example
+This example highlights the approach mentioned for transacting payments using operations and assets and using the [Stellar RPC](../../../data/apis/rpc/README.mdx) with [Stellar Client SDKs](/docs/tools/sdks/client-sdks#overview) to perform all actions needed. 
 
 ### Send a Payment
-
-When sending payments on Stellar, you build a transaction with a payment operation, sign it, and submit it to the network. Stellar stores and communicates transaction data in a binary format called [XDR](../../../learn/fundamentals/data-format/xdr.mdx), which is optimized for network performance but unreadable to the human eye. Luckily, [Horizon API](../../../data/apis/horizon/README.mdx), [Stellar RPC](../../../data/apis/rpc/README.mdx), and the [Stellar SDKs](../../../tools/sdks/README.mdx) convert XDRs into friendlier formats. Here’s how you might send 10 lumens to an account:
-
-#### Using the Stellar RPC
-
+Lets demonstrate a payment of an asset on Stellar. We will build a transaction with a payment operation to send 10 Lummens from a sender account to a receiver account, sign it as the sender account, and submit it to the network.
+- Submitting the transaction to the SDF-maintained public testnet instance of Stellar RPC server. 
+- When submitting transactions to the RPC server it's possible that you will not receive a response from the server due to network conditions. 
```

Comment:
Worth cross-referencing [this guide](https://developers.stellar.org/docs/data/apis/horizon/api-reference/errors/error-handling#error-handling-for-transaction-submissions) here as there are many such cases.